core/src/main/resources/error/error-classes.json

```diff
@@ -290,6 +290,46 @@
           "Null typed values cannot be used as arguments of <functionName>."
         ]
       },
+      "NUMBER_FORMAT_CURRENCY_MUST_BEFORE_DECIMAL" : {
+        "message" : [
+          "Currency characters must appear before any decimal point in the number format: '<numberFormat>'"
+        ]
+      },
+      "NUMBER_FORMAT_CURRENCY_MUST_BEFORE_DIGIT" : {
+        "message" : [
+          "Currency characters must appear before digits in the number format: '<numberFormat>'"
+        ]
+      },
+      "NUMBER_FORMAT_EMPTY" : {
+        "message" : [
+          "The format string cannot be empty"
+        ]
+      },
+      "NUMBER_FORMAT_THOUSANDS_SEPARATOR_MUST_BEFORE_DECIMAL" : {
+        "message" : [
+          "Thousands separators (, or G) may not appear after the decimal point in the number format: '<numberFormat>'"
+        ]
+      },
+      "NUMBER_FORMAT_THOUSANDS_SEPARATOR_MUST_HAVE_DIGIT_IN_BETWEEN" : {
```

Comment:
The name is too long. Is it possible to make it slightly shorter?

Reply:
Change it to 'NUMBER_FORMAT_SEPARATOR_MUST_HAVE_DIGIT_BETWEEN'?
It seems that the meaning of this error cannot be well displayed.

Reply:
Done